sources/reference/admiral.md

```diff
@@ -90,7 +107,23 @@ Confirm that the required values have been entered correctly. If you have decide
 
 Admiral will then install dependencies, pull Docker images, and install the database. This may take several minutes. When it is complete, the location of the Admiral UI and the login token are printed. Navigate to the UI location in a web browser and enter the given access key to access the installer UI. The rest of the services will be started from there.
 
-### Info
+### upgrade
+This command will upgrade the Shippable installation. To list the available
+versions, run `git tag` command inside `admiral` directory. To install
+Shippable version corresponding to any tag, run `git checkout <tag>` and then
+run `./admiral.sh upgrade` command.
+All the Shippable services will be upgraded to the new tag once the command is
+completed.
+
+### restart
+This is a helper command to recover from unexpected machine shutdown or
+reboots. Even if one(or more) Shippable services go down because of any reason,
```

Comment:
Missing a space after "one."  And I would remove "even" because this command will typically by run in that case; it's not an usual case where it will also work.  And perhaps "reboot" could be singular.  It's strange that shutdown and reboots don't match.